src/Accelerators/NNPA/Conversion/ONNXToZHigh/ONNXLegalityCheck.cpp

```diff
@@ -848,6 +849,10 @@ bool isSuitableForZDNN<ONNXConvOp>(ONNXConvOp op) {
   if (!isWOK)
     return false;
 
+  // Workaround for issue #1517 Accuracy issue  at C != 1, kH = 1, kW=1
+  if (inputShapeC > 1 && kernelShapeH == 1 && kernelShapeW == 1)
```

Comment:
You may exclude C == -1 as well for unknown dimensions.

Reply:
Done. Thanks!